README.md

```diff
@@ -218,19 +346,49 @@ Debug
 
 To access the console, you can use `shell_plus`:
 
-```
-$ make shell
+    make shell
+
+
+Provisioning sandboxes
+----------------------
+
+When configured correctly, the instance manager will automatically provision
+sandboxes whenever a pull request is made on GitHub. If you want to provision a
```

Comment:
@omarkhan "whenever a pull request is made on GitHub".. append "on your configured ``WATCH_FORK``, by members of your configured ``WATCH_ORGANIZATION``" ?